.github/workflows/periodic-reporter.yaml

```diff
@@ -0,0 +1,147 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'reporter'
+
+on:
+  schedule:
+    # 2 hours after scheduled periodic and once again in the evening
+    - cron: '0 5,17 * * *'
+  workflow_dispatch:
+
+jobs:
+  report:
+    permissions:
+      issues: 'write'
+
+    runs-on: 'ubuntu-latest'
+
+    steps:
+      - uses: 'actions/github-script@v6'
+        with:
+          script: |-
+                  // label for all issues opened by reporter
+                  const periodicLabel = 'periodic-failure';
+
+                  // check if any reporter opened any issues previously
+                  const prevIssues = await github.paginate(github.rest.issues.listForRepo, {
+                    owner: 'GoogleCloudPlatform',
+                    repo: 'terraform-google-secure-cicd',
+                    state: 'open',
+                    labels: [periodicLabel]
```

Comment:
For the future, might want to consider filtering on `user.login` and also `title` to minimize someone adding the label to a issue and it being returned.